script/profile_db_generator.rb

```diff
@@ -90,38 +83,58 @@ def ensure_perf_test_topic_has_right_title!
   unbundled_require 'gabbler'
 end
 
-puts "Creating 100 users"
-users = 100.times.map do |i|
+number_of_users = 100
+puts "Creating #{number_of_users} users"
+number_of_users.times.map do |i|
   putc "."
-  create_admin(i)
```

Comment:
Changing this to just normal user so that it is more realistic and also so that we have a normal user to benchmark against.